### DISCUSSION TITLE
TumblThree 2.10.2 released!

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#441] [Announcements]
### TumblThree 2.10.2

- Bug fixes
  - Correct filename extension for pnj links
  - Fix shown progress messages
  - Some Tumblr items are skipped due to an error while crawling
  - Some Tumblr images aren't downloaded due to an error determining original size
  - An error message is formatted wrong
  - Sometimes automatic blog save timer event occurs after timer is destroyed

<hr /><em>This discussion was created from the release <a href='https://github.com/TumblThreeApp/TumblThree/releases/tag/v2.10.2'>TumblThree-v2.10.2</a>.</em>

**Comment by tumblrdrei** [comment]
I haven't found any place where I can inform that Twitter blogs are not working anymore. It's not downloading anything. In the previous release it was also not working. I guess you're probably working on a solution, but I just wanted to inform you about it. 

You do a great work. 
Thank you
